uniffi_bindgen/src/bindings/swift/templates/RustBufferHelper.swift

```diff
@@ -130,16 +130,16 @@ protocol Serializable {
 // Types confirming to `ViaFfi` can be transferred back-and-for over the FFI.
 // This is analogous to the Rust trait of the same name.
 protocol ViaFfi: Serializable {
-    associatedtype Value
-    static func lift(_ v: Value) throws -> Self
-    func lower() -> Value
+    associatedtype FfiType
```

Comment:
It's unfortunate, but in the `extension Dictionary` below the Swift compiler couldn't disambiguate between `ViaFfi.Value` and `Dictionary.Value` in the `where` declaration